css/amp-story-player-iframe.css

```diff
@@ -37,6 +37,7 @@ main {
   height: 100%;
   position: relative;
   overflow: hidden;
+  transform-style: preserve-3d;
```

Comment:
https://stackoverflow.com/questions/17977220/can-css3-translatez-be-used-instead-of-z-index